app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :created_events, class_name: 'Event', foreign_key: 'user_id', dependent: :destroy
```

Comment:
#### Associations

##### User as event creator

* [ ] User model: `has_many :events, foreign_key: :creator_id` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- I see you have user_id instead of creator_id but please change it, the reason to change this is to practice creating foreign keys with a different name than the default one.